kubeflow/core/all.libsonnet

```diff
@@ -3,13 +3,11 @@
     local ambassador = import "kubeflow/core/ambassador.libsonnet",
     local jupyterhub = import "kubeflow/core/jupyterhub.libsonnet",
     local nfs = import "kubeflow/core/nfs.libsonnet",
-    local tfjob = import "kubeflow/core/tf-job.libsonnet",
     local spartakus = import "kubeflow/core/spartakus.libsonnet",
     local centraldashboard = import "kubeflow/core/centraldashboard.libsonnet",
     local version = import "kubeflow/core/version.libsonnet",
 
     all:: jupyterhub.all(params)
-          + tfjob.all(params)
           + ambassador.all(params)
```

Comment:
I think the all prototype should continue to install the TFJob operator. This way folks who want all the components can install them with a single package. But we can also have a prototype for just TFJob.